spec/helpers/models_helper.rb

```diff
@@ -0,0 +1,34 @@
+def create_and_activate_user(name)
+  user = User.create!(
+    name:, email: "#{name}@example.com", password: '123456', bio: "Bio of #{name}", photo: "#{name}.jpg"
+  )
+  user.confirm
+  user.save
+  user
+end
+
+def create_posts_for_user(user, count: 1)
+  posts = []
+  count.times do |i|
+    posts << Post.create!(
+      user:,
+      title: "Post #{i}",
+      text: "This is body of post #{i}",
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+  posts
+end
+
+def create_comments_for_post_by_user(post, user, count: 1)
+  comments = []
+  count.times do |i|
+    comments << Comment.create!(
+      user:,
+      post:,
```

Comment:
- [ ] We need the `user` to add a comment to a `post`. You didn't include the user and the post.


  I think all of these contribute to those syntax errors.

  Kindly resolves these issues.

Reply:
When we pass `user:user, post:post` rubocop asks us to omit values.